scripts/microbe_stage/microbe_operations.as

```diff
@@ -895,29 +895,31 @@ bool validPlacement(CellStageWorld@ world, ObjectID microbeEntity, const Organel
     return touching;
 }
 
-// speciesName decides the template to use, while individualName is
-// used for referencing the instance
 ObjectID spawnMicrobe(CellStageWorld@ world, Float3 pos, const string &in speciesName,
-    bool aiControlled)
+    bool aiControlled, bool partOfColony)
 {
     assert(world !is null);
     assert(speciesName != "");
 
     if(pos.Y != 0)
         LOG_WARNING("spawnMicrobe: spawning at y-coordinate: " + pos.Y);
 
-
     // Create microbeEntity with correct template, physics and species name
-    auto microbeEntity = _createMicrobeEntity(world, aiControlled, speciesName,
-        // in_editor
-        false);
+    auto microbeEntity = _createMicrobeEntity(world, aiControlled, speciesName, false);
```

Comment:
The comment should be preserved here. Anyway I don't think the parameter is used anymore, but as long as it exists, the comment should be kept.